linkedin_matrix/db/upgrade/v10_http_header_table.py

```diff
@@ -0,0 +1,18 @@
+from mautrix.util.async_db import Connection
+
+from . import upgrade_table
+
+
+@upgrade_table.register(description="Add a header table for storing all of the headers")
+async def upgrade_v10(conn: Connection):
+    await conn.execute(
+        """
+        CREATE TABLE http_header (
+            mxid  TEXT,
```

Comment:
probably it's worth having a foreign key for mxid to the users table, right?

Reply:
I copied what I saw in the cookies table, should I add one there as well?

Reply:
This is probably fine. We can have the constraint later.